templates/tpos/index.html

```diff
@@ -655,19 +655,32 @@ <h6 class="text-subtitle1 q-my-none">
       new Date(obj.time * 1000),
       'YYYY-MM-DD HH:mm'
     )
-    obj.tpos = ['/tpos/', obj.id].join('')
-    obj.shareUrl = [
-      window.location.protocol,
-      '//',
-      window.location.host,
-      obj.tpos
-    ].join('')
+    obj.tpos = `/tpos/${obj.id}`
+    obj.shareUrl = `${window.location.protocol}//${window.location.host}${obj.tpos}`
+
     obj.items = JSON.parse(obj.items)
-    obj.itemsMap = new Map()
-    obj.items.forEach((item, idx) => {
-      let id = `${obj.id}:${idx + 1}`
-      obj.itemsMap.set(id, {...item, id})
-    })
+    obj.itemsMap = new Map(
+      obj.items.map((item, idx) => {
+        const id = `${obj.id}:${idx + 1}`
+        return [id, {...item, id}]
+      })
+    )
+
+    // Retrieve theme, dark mode, and gradient settings from localStorage.
+    const getSetting = key =>
+      localStorage.getItem(`lnbits.${key}`)?.split('|')[1] || null
```

Comment:
why not use `this.$q.localStorage.getItem()`?
the `split('|')[1]` goes into the internals of quasar.